### ISSUE TITLE
Training loss on custom dataset

### ISSUE THREAD

**Issue by mmmcn** [zju3dv/neuralbody#90]
Thanks for sharing this excellent work!  
I am trying to train my own monocular dataset following your instruction, but I found that my training loss was around 0.03 most of the time, I tried to adjust the learning rate, but it did not  seem to work. I only revised the image resolution in config file according to my dataset, could you provide  some advice to me, thanks in advance.

**Comment by pengsida** [comment]
You need to ensure that the camera poses are correct.
